README.md

```diff
@@ -1 +1,26 @@
-# Youtube
+# Youtube page 
+> This project consists of building an HTML document that matches the appearance of youtube video player page.
+
+# Build by
+- HTML
+- CSS
+
+
+## Live Demo
+
+[Live Demo Link](https://rawcdn.githack.com/dasileker/Youtube/f4758154d538e17becf46387252db440d5e24384/youtube.html)
+
+
+
+**Author 1**
+GitHub [@dasileker](https://github.com/dasileker)
+
+**Author 2**
+GitHub [@olyadi](https://github.com/CornerstoneII)
```

Comment:
Add other social media profile links like twitter and linkeding

Reply:
done